tests/src/test/scala/whisk/core/database/test/DbUtils.scala

```diff
@@ -45,13 +44,14 @@ import whisk.core.entity.types.EntityStore
  * operations with those that flow through the cache. To mitigate this, use unique asset
  * names in tests, and defer all cleanup to the end of a test suite.
  */
-trait DbUtils extends TransactionCounter {
+trait DbUtils {
   implicit val dbOpTimeout = 15 seconds
-  override val instanceOrdinal = 0
-  val instance = InstanceId(instanceOrdinal)
+  val instance = InstanceId(0)
   val docsToDelete = ListBuffer[(ArtifactStore[_], DocInfo)]()
   case class RetryOp() extends Throwable
 
+  def transid() = TransactionId.testing
```

Comment:
As noted earlier a sequential counter here would be useful when looking at a test log.